### PR TITLE
Moves brainwash surgery from role restricted to devices and tools in the uplink

### PR DESCRIPTION
**Pull request by ForrestWick**
## About The Pull Request

Removes role restrictions on brainwash surgery and allows traitors to buy it alongside the surgery kits under the tools category.

## Why It's Good For The Game

Maroon objectives pretty much boil down to killing someone 90% of the time and this could allow an interesting alternative, alongside allowing some neat gimmicks as well and for sec to have an alternative to execution or permabrig. Honestly, there's not much of a point to keeping it locked behind medical doctor anyway since nobody who plays a med traitor will buy it.

## Changelog
Moved brainwash surgery disk from role restricted to devices and tools.
/:cl:

### CODE REVIEW

**Comment by kappa-sama** [comment]
pr broken revert first commit

**Comment by ForrestWick** [comment]
> pr broken revert first commit

The change functions correctly when tested. Travis is just shitting the bed

**Comment by kappa-sama** [comment]
that's why

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1719,14 +1726,6 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 	cost = 12
 	restricted_roles = list("Research Director", "Scientist", "Roboticist")
 
-/datum/uplink_item/role_restricted/brainwash_disk
```
Comment (ForrestWick) [inline code/modules/uplink/uplink_items.dm]: .

**Comment by GrayRachnid** [comment]
I'm against this PR because it's locked behind doctors for good reason. Reason being that it'd be stronger on other roles like engineer or an assistant who aren't expected to be doing surgeries.

**Comment by necromanceranne** [comment]
These roles are also more easily able to make hideaways to perform the surgery in. Which is a fairly big deal given how medical doctors need to made do with working their own department and potentially under scrutiny.

**Comment by ForrestWick** [comment]
> These roles are also more easily able to make hideaways to perform the surgery in. Which is a fairly big deal given how medical doctors need to made do with working their own department and potentially under scrutiny.



> I'm against this PR because it's locked behind doctors for good reason. Reason being that it'd be stronger on other roles like engineer or an assistant who aren't expected to be doing surgeries.

You say this while holodeck surgery exists, maint access is given like candy, wholeass job changes given like candy, maint surgeries exist, etc. Even then it isnt all that powerful because even medical doctors have to silence the victim prior of they're fucked, and you can't crit someone because they'll die during surgery if you do

**Comment by necromanceranne** [comment]
They add variables that the med doctor might not be able to circumvent. And no, it is somewhat easier for them because they can more easily excuse why they might be creating a discreet and remote construction in maint. Engineers and assistants have an ic excuse to be building things, and engineers have ready, round start access to the materials and tools.

**Comment by ForrestWick** [comment]
> They add variables that the med doctor might not be able to circumvent. And no, it is somewhat easier for them because they can more easily excuse why they might be creating a discreet and remote construction in maint. Engineers and assistants have an ic excuse to be building things, and engineers have ready, round start access to the materials and tools.
Literally anyone in any job can and will set up.building projects. Hell I see people with chem labs in the bar or library pretty often. I even see assistants set up back alley surgery stuff and nobody bats at eye. Its all seen as normal behavior until you force surgeries on people

**Comment by GrayRachnid** [comment]
The only people i've seen with chemlabs in the bar and library are you 😂

**Comment by ForrestWick** [comment]
> The only people i've seen with chemlabs in the bar and library are you

I mean sure, once or twice so I could play with SDGF. But I've seen others do it

**Comment by necromanceranne** [comment]
My usual chemlab is in a mining pod.

If someone didn't steal it for a fuckpad.

**Pull request merged**